pkg/test/framework/components/policybackend/policybackend.go

```diff
@@ -197,9 +198,22 @@ func (c *kubeComponent) Init(ctx environment.ComponentContext, deps map[dependen
 	addressInCluster := fmt.Sprintf("%s:%d", svc.Spec.ClusterIP, svc.Spec.Ports[0].TargetPort.IntVal)
 	scope.Debugf("Policy Backend in-cluster address: %s", addressInCluster)
 
-	forwarder := kube.NewPortForwarder(ctx.Settings().KubeConfig, pod.Namespace, pod.Name, int(svc.Spec.Ports[0].TargetPort.IntVal))
-	if err = forwarder.Start(); err != nil {
-		return nil, err
+	rc, config, err := kube.NewRestClient(ctx.Settings().KubeConfig, "")
```

Comment:
There's a lot of copy & paste here .... could we make a utility to help here?

Reply:
I meant to, but I can't decide where to place the common func, any suggestions?